plutus-benchmark/flat/Dataset.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
I also thought it would be nice to use the data samples you provided. However, I got no feedback on that idea, so I assumed it's tanked? @michaelpj do you think I should add any new issues for these development ideas?

Reply:
I don't know how easy it would be to use the scripts in `data`. If it's easy, I think we should do that.

Reply:
We may end up going back the other way and having those be generated later - mainly I think it'll be a pain to have *two* ways of doing it.

Reply:
I think it's good to have two ways of doing it because in some circumstances we'll want the output of the current version of the compiler and in others (like benchmarking evaluators) we'll want fixed scripts that don't change for extraneous reasons.

To see how to use the files in `data`, see `loadScript` in `getAppliedScript` in `validation/Main.hs`.  An objection to doing that is that you have to have certain filenames coded into the source.